pkg/schema/load/generic.go

```diff
@@ -88,10 +89,38 @@ func buildGenericScuemata(famval cue.Value) (schema.VersionedCueSchema, error) {
 		}
 		major++
 	}
-
+	if lastgvs.actual, err = prepare(lastgvs.actual); err != nil {
+		return nil, err
+	}
 	return first, nil
 }
 
+func prepare(iCUE cue.Value) (cue.Value, error) {
+	iter, err := iCUE.Fields(cue.Definitions(true))
+	if err != nil {
+		return iCUE, err
+	}
+	for iter.Next() {
+		a := iter.Value().Attribute("cuetsy")
+		_, found, err := a.Lookup(0, "targetType")
+		if a.Err() != nil || !found || err != nil {
+			inst, err := rt.Compile("intermedia", `_,@cuetsy(targetType="interface")`)
+			if err != nil {
+				return iCUE, err
+			}
+			iCUE = iCUE.FillPath(cue.ParsePath(iter.Label()), inst.Value())
+			b, _ := iter.Value().Unify(inst.Value()).MarshalJSON()
+			fmt.Println("<<<<<<<<<", string(b))
```

Comment:
Attributes are not a part of the data model, so this will never show you anything. If you want to inspect the effects of your `FillPath()`, you'll need to do something more like this:

```cue
for _, a := range iter.Value().Attributes(cue.ValueAttr) {
    fmt.Println(a.Name(), a.Contents())
}
```